parking_permits/models/parking_zone.py

```diff
@@ -24,9 +21,6 @@ class ParkingZone(TimestampedModelMixin, UUIDPrimaryKeyMixin):
     name = models.CharField(_("Name"), max_length=128)
     description = models.TextField(_("Description"), blank=True)
     description_sv = models.TextField(_("Description sv"), blank=True)
-    shared_product_id = models.UUIDField(
```

Comment:
Should we do this modification in future PR as at this point it will break talpa integration